packages/slice-machine/src/features/customTypes/customTypesTable/CustomTypesTable.tsx

```diff
@@ -82,49 +87,80 @@ export const CustomTypesTable: FC<CustomTypesTableProps> = ({
     );
   }
 
+  // The wrapping `<div>` prevents `<Table>` from growing vertically.
   return (
-    <Table>
-      <TableHead>
-        <TableRow>
-          <TableCell>
-            <Icon name="notes" size="medium" />
-          </TableCell>
-          <TableCell>Label</TableCell>
-          <TableCell>API ID</TableCell>
-          <TableCell>Limit</TableCell>
-          <TableCell />
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {sortedCustomTypes.map((customType: CustomType) => {
-          const { repeatable, label, id } = customType;
+    <div>
```

Comment:
I was surprised by that as well. There must be a reason why it has that by default, but it would be great to remove it. For now, the extra `<div>` prevents `Table` from growing.

Reply:
Arf, not super great true, did you asked why to editor @angeloashmore?

Anyway, should not block a merge we can quickly remove it the day they remove it also